src/test/ui/const-eval/const_panic_libcore.stderr

```diff
@@ -0,0 +1,33 @@
+error: this constant cannot be used
+  --> $DIR/const_panic_libcore.rs:14:1
+   |
+LL | const Z: () = panic!("cheese");
+   | ^^^^^^^^^^^^^^----------------^
+   |               |
+   |               evaluated panic invocation 'cheese', $DIR/const_panic_libcore.rs:14:15
```

Comment:
"evaluated" doesn't feel right in user output. Maybe mimick the runtime output and say:
```
panicked at 'cheese', $DIR/const_panic_libcore.rs:14:15
```

Reply:
I had that. Our test suite is not able to cope with that because it greps for `panicked at` and reports a compiler panic. Once the test suite is adjusted, I can fix the message

Reply:
Maybe something like `const panic at 'cheese'` instead?

Reply:
Why not just change the test suite to search for `' panicked at '`, i.e. take advantage of the fact that there should be a non-space character before the space before "panicked"?

Reply:
Once https://github.com/rust-lang/rust/pull/52197 is merged we don't need any more text hacks. I'll just wait for that